images/ceph/Dockerfile

```diff
@@ -19,6 +19,9 @@ FROM BASEIMAGE
 ARG S5CMD_VERSION
 ARG S5CMD_ARCH
 
+# install 'ip' tool for Multus
+RUN dnf install -y iproute
```

Comment:
Will this effectively be a no-op for now since it's included in the base image? Or should we add this later when it actually gets removed from the base image?

Reply:
I think it will be a noop today. I thought it'd probably be safest to add this now so it doesn't cause unnecessary headache in the future, but I'm open to either.

Reply:
In a local build of this change, the uncompressed image size increased from 1.3GB now to 1.48GB. Shall we either hold off on this change or understand how to trim this? It seems like excessive growth for a simple tool.

Reply:
@travisn  wrote:
> In a local build of this change, the uncompressed image size increased from 1.3GB now to 1.48GB. Shall we either hold off on this change or understand how to trim this? It seems like excessive growth for a simple tool.

I can confirm the effect: For me, the image size increased locally from 1.33 GB to 1.55 GB.

While I do agree with the general spirit of this PR to move the dependency for getting the ip command from the ceph base image to the rook image itself, I also agree with @travisn's suggestion to hold this change off until we have better understood and hopefully undonde the growth effect.

Reply:
This is fixed now with `dnf clean all`. The extra content was likely the repo metadata files that are added when installing a package. I also updated it not to pull in "weak" dependencies which could help.

Reply:
yes and for me the size increased by around  40 MB only from 1.24GB to 1.28GB